src/main/resources/db/migration/V1__Initial_Tables.sql

```diff
@@ -0,0 +1,48 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE TABLE IF NOT EXISTS beacons (
+    id uuid PRIMARY KEY,
+    beacon_type text,
+    hex_id text,
+    manufacturer text,
+    model text,
+    serial_number text,
+    battery_expiry TIMESTAMP,
+    last_serviced TIMESTAMP
+);
+CREATE TABLE IF NOT EXISTS person (
+    id uuid PRIMARY KEY,
+    person_type text,
+    name text,
+    telephone text,
+    email_address text,
+    address_line_1 text,
+    address_line_2 text,
+    address_line_3 text,
+    address_line_4 text,
+    address_line_5 text,
+    address_line_6 text,
+    address_line_7 text
```

Comment:
We lifted this from GOV.UK Notify [docs](https://docs.notifications.service.gov.uk/java.html#send-a-letter-arguments-templateid-required).  Whilst it will allow for any international address, it would mean MCA can't easily do things like mapping beacon owners or exchange data with other govt departments (e.g. a local authority wants anonymised data on seafarers within its remit).

We should instead capture at least `country` and `postcode`, as well as something to represent `city`/`locality`.

I found a few examples of schemas to capture international addresses:
- Schema.org's [PostalAddress type](https://schema.org/PostalAddress) ("Founded by Google, Microsoft, Yahoo and Yandex, Schema.org vocabularies are developed by an open community process, using the public-schemaorg@w3.org mailing list and through GitHub.")
- Shopify's [@shopify/address](https://github.com/Shopify/quilt/blob/master/packages/address/package.json) node package (we don't have to use Shopify's package, just copy its schema)
- The Android SDK's [Address class](https://developer.android.com/reference/android/location/Address) (perhaps more comprehensive than we need).
- I couldn't find GOV.UK / GDS guidance on persisting addresses 😞 

Would something like this work for the Beacons use case? (Mostly adapted from the schema.org example)
```sql
addressLines text[],   /* Street / building name etc. [0] */
addressLocality text,  /* City / town / village, .e.g "Falmouth" */
addressRegion text,  /* State / Province / County, e.g. "Cornwall" */
addressCountry text(2), /* [ISO 3166-1](https://en.wikipedia.org/wiki/ISO_3166-1) two-letter country code */
postalCode text, /* Postal / Zip code */
```

---
[0] [] = [Array datatype](https://www.postgresql.org/docs/9.1/arrays.html), I don't think the street name / building name etc. need to be normalised.  We do need more than 2-3 lines here considering edge cases, e.g. `Zack Adlington, MV Cest la Vie, c/o Harbourmaster, Djibouti Yacht Pontoon, 13 Independence Street  ...`

Reply:
@zackads Agree with the above, definitely worth a discussion. Would you be happy to capture this as an issue on the repository and we can look to merge this merge request?

We can then further migration scripts on top of this/update the POJOs to be inline with the above in the next merge request.